### ISSUE TITLE
aws_autoscaling_group: 'terraform plan' does not notice changes to termination_policies

### ISSUE THREAD

**Issue by patricklucas** [hashicorp/terraform#3052]
If you specify `termination_policies` on an `aws_autoscaling_group`, then change its value in the console, the update is not reflected during `terraform plan`.

**Steps to reproduce:**

1. Create an ASG:

    ```
    resource "aws_autoscaling_group" "my-asg" {
        ...
        termination_policies = ["OldestInstance"]
    }
```
2. `terraform apply`
3. In the console, change the ASG's termination policies to "NewestInstance"
4. `terraform plan`

**Expected behavior:** Terraform notices the change and plans to change `termination_policies` back to "OldestInstance".

**Actual behavior:** Terraform thinks that `termination_policies` remains "OldestInstance" and plans to take no action.

**Comment by catsby** [comment]
Thanks for opening this! I'll make sure it gets addressed in https://github.com/hashicorp/terraform/pull/2890

**Comment by stack72** [comment]
@catsby / @jen20 I believe this can now be closed due to the merging of #2890

**Comment by catsby** [comment]
Thanks!
